fix typo in string search api docs

ashleysommer opened the pull request:
findlast cannot be called with 3 arguments, (with the third being a a start index), this should read `findprev(`

ashleysommer committed:
fix typo in documentation

findlast cannot be called with 3 arguments, (with the third being a a start index), this is mean to read `findprev(`

ararslan reviewed (approved):
Good catch, thanks!

fredrikekre commented on base/strings/search.jl:
```diff
@@ -259,7 +259,7 @@ findnext(t::AbstractString, s::AbstractString, i::Integer) = _search(s, t, i)
 
 Find the last occurrence of `pattern` in `string`. Equivalent to
-[`findlast(pattern, string, lastindex(s))`](@ref).
+[`findprev(pattern, string, lastindex(s))`](@ref).
```
Would also be a good idea to update to `lastindex(string)` instead, to match the signature on the lines above:

```suggestion
[`findprev(pattern, string, lastindex(string))`](@ref).
```
alternatively change `string` to `s` on L259.

fredrikekre committed:
Update base/strings/search.jl

Change variable name in example from `s` to `string` to match the signature on the lines above it.

Co-Authored-By: ashleysommer <ashleysommer@gmail.com>

fredrikekre commented:
Thanks, and welcome as a contributor!
